sagemaker_batch_transform/tensorflow_open-images_tfrecord/tensorflow-serving-tfrecord.cli.ipynb

```diff
@@ -45,7 +45,7 @@
     "    'cn-north-1': 422961961927,\n",
     "    'cn-northwest-1': 423003514399, \n",
     "}  \n",
-    "if region == 'cn-north-1' or 'cn-northwest-1':\n",
+    "if region in ['cn-north-1', 'cn-northwest-1']:\n",
```

Comment:
In this case you may want to leverage the map you already have, just a suggestion.

```
if region in account_id_cn:
    ...
```

Reply:
(Same for other occurrences)